ProcessLib/LiquidFlow/LiquidFlowLocalAssembler.h

```diff
@@ -86,6 +86,12 @@ class LiquidFlowLocalAssembler : public LiquidFlowLocalAssemblerInterface
                   std::vector<double>& local_K_data,
                   std::vector<double>& local_b_data) override;
 
+    void coupling_assemble(double const t, std::vector<double> const& local_x,
```

Comment:
Another PR is fine. We should keep the so far consistent functions/variables naming.